index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Surf and Paddle | An Awesome Blog of Awesomeness</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="./main.css" />
+    <link href='http://fonts.googleapis.com/css?family=Muli:300' rel='stylesheet' type='text/css' />
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" />
+  </head>
+  <body>
+    <div class="header__background">
+      <div class="header__overlay">
+        <nav>
+          <ul class="header__nav">
+            <li><a href="./index.html">About</a></li>
+            <li><a href="./index.html">Search<i class="fa fa-search fa-rotate-90"></i></a></li>
```

Comment:
From now on, when linking, you needn't use the `.` in front of the `/`. This is because we will be using web servers, and they deliver content differently